.gitmodules

```diff
@@ -7,3 +7,12 @@
 [submodule "third_party/eigen"]
 	path = cpp/thirdparty/eigen
 	url = https://github.com/eigen-mirror/eigen.git
+[submodule "vcpkg"]
+	path = vcpkg
+	url = https://github.com/Microsoft/vcpkg.git
+[submodule "external/CLI11"]
+	path = external/CLI11
+	url = git@github.com:CLIUtils/CLI11.git
+[submodule "external/Eigen3"]
+	path = external/Eigen3
+	url = https://gitlab.com/libeigen/eigen.git
```

Comment:
Can we use `vcpkg` for these too?

Reply:
For sure -- this was a demonstration of temporarily using development versions of these libraries. To switch back to the vcpkg-provided versions, you just remove the library name from the `FARM_NG_PROVIDER_DEV_PACKAGES` cache variable in `CMakePresets.json`